src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Anonymize/Anonymize.ts

```diff
@@ -46,6 +47,7 @@ export var anonymizeDirective = (
                 ]).then((args) => {
                     scope.isOptional = args[1];
                     scope.data.model = scope.isOptional ? adhUser.data.anonymize : false;
+                    scope.data.model = scope.localDefault || scope.data.model;
```

Comment:
funny. trying out your code literally produced strange errors because `default` is a keyword.